python/README.md

```diff
@@ -1,26 +1,26 @@
-# Overview
-The purpose of the worker is to carry out data analysis tasks, for example computing an embedding. Data analysis tasks get received from a SQS queue and results will get submitted to a SNS topic the API is subscribed to.
+worker-python
+=============
 
+Overview
+--------
+The purpose of the worker is to carry out data analysis tasks. `worker-python` is the main entry point
+for data analysis tasks. Each bioinformatics task to be performed is received from an AWS SQS queue.
+Results are submitted to an SNS topic that is listened to by the `api` module.
 
-# Build and Deploy step
-The deployment step for this repository consits of deploying all resources needed for work to be carried out. Note that the deployment step in this repository does not start the work. Work gets started dynamically upon a user request by the [API](https://github.com/biomage-ltd/api).
+Running the worker
+------------------
 
-CI deployment needs four variables that need to be created on the CI/CD side:
+### Start the worker
 
-* `GITLAB_DEPLOY_USER` and `GITLAB_DEPLOY_PASSWORD`. These need to specify a valid GitLab deploy key that has read rights to the Docker registry. This secret is used by dynamically generated jobs to automatically be able to pull the worker image.
-* `K8S_SECRET_AWS_ACCESS_KEY_ID`, `K8S_SERET_AWS_SECRET_ACCESS_KEY`, `K8S_SECRET_AWS_DEFAULT_REGION`, which specify the IAM account and default region to be used with this worker.
+See the main README for instructions on how to run the workers in Docker.
 
-# Development
+### Process tasks
+Tasks are automatically processed when they are received from the SQS queue specified.
 
-## Run locally
+For **local development**, make sure you have InfraMock running alongside the `ui` and `api`. Tasks should
```

Comment:
put links to inframock, the api and the ui and tell them to see their readmes on how to set them up

Reply:
Done.